tests/components/darksky/test_sensor.py

```diff
@@ -195,3 +195,7 @@ def test_setup(self, mock_req, mock_get_forecast):
         assert state.attributes.get("friendly_name") == "Dark Sky Summary"
         state = self.hass.states.get("sensor.dark_sky_alerts")
         assert state.state == "2"
+
+        state = self.hass.states.get("sensor.dark_sky_daytime_high_temperature_1d")
+        assert state is not None
+        assert state.attributes.get("device_class") == "temperature"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5518609)